src/System.Threading.Timer/tests/TimerFiringTests.cs

```diff
@@ -122,8 +120,8 @@ public void Timer_CanDisposeSelfInCallback()
             t.Dispose();
             are.Set();
         });
-        t = new Timer(tc, null, 1, -1);
-        Assert.True(are.WaitOne(50));
+        t = new Timer(tc, null, 10, -1);
```

Comment:
Changing this from 1 to 10 could actually make things worse.  I'd leave it at 1.  If this test is going to fail, it's because the callback didn't complete within the time that we're waiting on the event... but making this dueTime larger actually delays when the timer invokes the callback. (In this case, the change is probably a nop due to rounding of timeout values.)